dbusmock/templates/urfkill.py

```diff
@@ -75,10 +75,14 @@ def block(self, index, should_block):
         new_block_state = 0
     if self.internal_states[objname] != new_block_state:
         path = '/org/freedesktop/URfkill/' + objname
-        obj = dbusmock.get_object(path)
-        self.internal_states[objname] = new_block_state
-        obj.Set('org.freedesktop.URfkill.Killswitch', 'state', new_block_state)
-        obj.EmitSignal('org.freedesktop.URfkill.Killswitch', 'StateChanged', '', [])
+        try:
+            obj = dbusmock.get_object(path)
+            self.internal_states[objname] = new_block_state
+            obj.Set('org.freedesktop.URfkill.Killswitch', 'state', new_block_state)
+            obj.EmitSignal('org.freedesktop.URfkill.Killswitch', 'StateChanged', '', [])
+        except:
```

Comment:
bare except: statements are utterly wrong. If there's an unexpected error in the calls, then it should be propagated as usual, and not papered over with a generic except:. That will also hide really bad stuff like SyntaxError or real programming errors. Please add a comment what the return value should mean, but I suppose it should just tell you whether the previous state was blocked or not? Determining that should work by looking at properties, not through exceptions.

Otherwise, please add a comment what could go wrong here, and only catch the expected exceptions.